lib/replicator.js

```diff
@@ -513,18 +513,18 @@ class Peer {
     else this._clearLocalRange(start, length)
 
     const i = Math.floor(start / DEFAULT_SEGMENT_SIZE)
-    const contig = this.core.header.hints.contiguousLength === this.core.state.length
+    const fullyContig = this.core.header.hints.contiguousLength === this.core.state.length
```

Comment:
(detail) unneeded renaming of the variable?

Reply:
The renaming was because initially I thought it meant that the local core was contiguous even if partially so. So renamed only to clarify that it is fully contiguous in the sense that it matches the latest length.

But yeah, no functionality change.